.circleci/config.yml

```diff
@@ -4,66 +4,140 @@
 
 version: 2
 jobs:
-  build:
+  compile:
     docker: [{ image: 'circleci/openjdk:8u171-node' }]
-    resource_class: xlarge
+    resource_class: large
```

Comment:
per https://circleci.com/docs/2.0/configuration-reference/#resource_class this is 4 vCPUs and 8GB RAM